CPAC/sca/sca.py

```diff
@@ -145,33 +143,8 @@ def create_sca(name_sca="sca"):
 
     concat.inputs.outputtype = "NIFTI_GZ"
 
-    # also write out volumes as individual files
-    # split = pe.Node(interface=fsl.Split(), name='split_raw_volumes_sca')
-    # split.inputs.dimension = 't'
-    # split.inputs.out_base_name = 'sca_'
-
-    # get_roi_num_list = pe.Node(util.Function(input_names=['timeseries_file',
-    #                                                      'prefix'],
-    #                                         output_names=['roi_list'],
-    #                                         function=get_roi_num_list),
-    #                           name='get_roi_num_list')
-    # get_roi_num_list.inputs.prefix = "sca"
-
-    # sca.connect(inputNode, 'timeseries_one_d', get_roi_num_list,
-    #            'timeseries_file')
-
-    # rename_rois = pe.MapNode(interface=util.Rename(), name='output_rois',
-    #                         iterfield=['in_file', 'format_string'])
-    # rename_rois.inputs.keep_ext = True
-
-    # sca.connect(split, 'out_files', rename_rois, 'in_file')
-    # sca.connect(get_roi_num_list, 'roi_list', rename_rois, 'format_string')
-
     sca.connect(corr, "out_file", concat, "in_files")
-    # sca.connect(concat, 'out_file', split, 'in_file')
     sca.connect(concat, "out_file", outputNode, "correlation_stack")
-    # sca.connect(rename_rois, 'out_file', outputNode,
-    #            'correlation_files')
```

Comment:
I think if we want to resurrect this functionality, we can rewrite it at least as easily as we could revive it from the commented-out code, so I deleted the commented-out code.